hack/e2e.sh

```diff
@@ -76,7 +78,7 @@ E2E_DOWN=""
 # The capd manager docker image name and tag. These have to be consistent with
 # what're used in capd default kustomize manifest.
 CAPD_DEFAULT_IMAGE="gcr.io/k8s-staging-cluster-api/capd-manager:dev"
-CAPD_IMAGE="gcr.io/k8s-staging-cluster-api/capd-manager:${CAPI_VERSION}"
+CAPD_IMAGE="gcr.io/k8s-staging-cluster-api/capd-manager:${CAPD_VERSION}"
```

Comment:
Is there any reason why CAPI_VERSION was used before for the CAPD_IMAGE?

Reply:
I just changed this part back. there seem to be an assumption that `CAPI_IMAGE` and `CAPD_IMAGE` always have the same tag in a release